tests/Microsoft.Identity.Web.Test/Microsoft.Identity.Web.Test.csproj

```diff
@@ -18,6 +18,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="CommunityToolkit.HighPerformance" Version="8.2.2" />
```

Comment:
?

Reply:
agree. this doesn't seem to be needed, as well as the playwright traces...I have several showing up in code flow.

Reply:
thanks it is not needed any longer, I toyed with some classes in there but decided not to use it